templates/statefulset.yaml

```diff
@@ -82,20 +86,43 @@ spec:
           value: "{{ .Values.messageBufferLen }}"
         - name: ACK_BUFFER_LENGTH
           value: "{{ .Values.ackBufferLen }}"
-        {{- if .Values.caCerts }}
+        - name: MIDDLEWARES_SOCKET
+          value: "passthrough:///unix://{{ $socket }}"
         volumeMounts:
```

Comment:
This is not technically correct, but we can afford to mount this even if not needed

Reply:
But you are right, this won't work without middlewares because there will be an error.

Reply:
Oh, I was referring to the `volumeMounts`, but I see: MIDDLEWARE_SOCKETS must be conditionally set